dsc/src/resolve.rs

```diff
@@ -51,74 +66,104 @@ pub fn get_contents(input: &str) -> Result<(Option<String>, String), String> {
         }
     };
 
-    let include_path = normalize_path(Path::new(&include.configuration_file))?;
+    let config_json = match include.configuration {
+        IncludeKind::FilePath(file_path) => {
+            let include_path = normalize_path(Path::new(&file_path))?;
 
-    // read the file specified in the Include input
-    let mut buffer: Vec<u8> = Vec::new();
-    match File::open(&include_path) {
-        Ok(mut file) => {
-            match file.read_to_end(&mut buffer) {
-                Ok(_) => (),
+            // read the file specified in the Include input
+            let mut buffer: Vec<u8> = Vec::new();
+            match File::open(&include_path) {
+                Ok(mut file) => {
+                    match file.read_to_end(&mut buffer) {
+                        Ok(_) => (),
+                        Err(err) => {
+                            return Err(format!("{} '{include_path:?}': {err}", t!("resolve.failedToReadFile")));
+                        }
+                    }
+                },
                 Err(err) => {
-                    return Err(format!("{} '{include_path:?}': {err}", t!("resolve.failedToReadFile")));
+                    return Err(format!("{} '{include_path:?}': {err}", t!("resolve.failedToOpenFile")));
                 }
             }
-        },
-        Err(err) => {
-            return Err(format!("{} '{include_path:?}': {err}", t!("resolve.failedToOpenFile")));
-        }
-    }
-    // convert the buffer to a string
-    let include_content = match String::from_utf8(buffer) {
-        Ok(input) => input,
-        Err(err) => {
-            return Err(format!("{} '{include_path:?}': {err}", t!("resolve.invalidFileContent")));
-        }
-    };
+            // convert the buffer to a string
+            let include_content = match String::from_utf8(buffer) {
+                Ok(input) => input,
+                Err(err) => {
+                    return Err(format!("{} '{include_path:?}': {err}", t!("resolve.invalidFileContent")));
+                }
+            };
 
-    // try to deserialize the Include content as YAML first
-    let configuration: Configuration = match serde_yaml::from_str(&include_content) {
-        Ok(configuration) => configuration,
-        Err(_err) => {
-            // if that fails, try to deserialize it as JSON
-            match serde_json::from_str(&include_content) {
-                Ok(configuration) => configuration,
+            match parse_input_to_json(&include_content) {
+                Ok(json) => json,
                 Err(err) => {
                     return Err(format!("{} '{include_path:?}': {err}", t!("resolve.invalidFile")));
                 }
             }
-        }
-    };
+            // // try to deserialize the Include content as YAML first
```

Comment:
here and below - why comment out this code instead of removing?

Reply:
this was accidentally left over from refactoring this code, will remove